sdk/communication/communication-network-traversal/test/public/utils/recordedClient.ts

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Context } from "mocha";
+import * as dotenv from "dotenv";
+
+import {
+  env,
+  Recorder,
+  record,
+  RecorderEnvironmentSetup,
+  isPlaybackMode
+} from "@azure/test-utils-recorder";
+import {
+  DefaultHttpClient,
+  HttpClient,
+  HttpOperationResponse,
+  isNode,
+  TokenCredential,
+  WebResourceLike
+} from "@azure/core-http";
+import { CommunicationRelayClient, CommunicationRelayClientOptions } from "../../../src";
+import { ClientSecretCredential, DefaultAzureCredential } from "@azure/identity";
+import { parseConnectionString } from "@azure/communication-common";
+
+if (isNode) {
+  dotenv.config();
+}
+
+export interface RecordedClient<T> {
+  client: T;
+  recorder: Recorder;
+}
+
+const replaceableVariables: { [k: string]: string } = {
+  COMMUNICATION_CONNECTION_STRING: "endpoint=https://endpoint/;accesskey=banana",
```

Comment:
I believe we've recently changed this in the pipeline from COMMUNICATION_CONNECTION_STRING
to either COMMUNICATION_LIVETEST_STATIC_CONNECTION_STRING or COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING. If this is similar to identity, COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING should suffice

Reply:
for samples and sample code i think we can leave it as COMMUNICATION_CONNECTION_STRING though